### DISCUSSION TITLE
DSLX variable

### DISCUSSION THREAD

**Discussion by Eirikr70** [immich-app/immich#3856] [Q&A] [answered]
Hello, my setup is totally functional, but I always have that log saying : `The "DSLX" variable is not set. Defaulting to a blank string.` Is it something that should be added in the `.env` file ?

**Comment by jrasm91** [comment]
Is this variable coming from your docker compose file? I do not believe it is an immich variable/setting/log.

**Reply by Eirikr70** [reply]
No, I don't have such a variable.

**Reply by jrasm91** [reply]
Do you have any references to DSLX in the compose file or `.env` file? Either as an variable name or value? The error "the variable is not set. defaulting to a blank string" is from docker compose. Specifically when an environment variable is referenced without a value, so it has to be coming from something in your compose file.

**Reply by Eirikr70** [reply, marked as answer]
No, I have no reference to a DSLX variable neither in the compose file nor in the .env file. I supposed it was something technical as appears here : https://google.github.io/xls/dslx_reference/
Anyway, it is no problem since everything works fine.
